### PR TITLE
Added Header Component!

### PR DESCRIPTION
**Pull request by mittalvansh**
Added Header Component!
Issue Resolved! Issue #1

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,13 +1,13 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
```
Comment (Waishnav) [inline src/index.css]: import all the font-weight of the Poppins. 
```@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');```

File: src/components/Header/Header.css [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+button .StarUs {
+    font-family: 'Cairo', sans-serif;
+    font-size: 1rem;
+    font-weight: 500;
+    margin-right: 1rem;
+}
+
+button .vertical {
+    border-left: 1px solid #5C5C5C;
```
Comment (Waishnav) [inline src/components/Header/Header.css]: ```border-left: 2px solid #5C5C5C``` would be great.
Also, add a hover-state to the button which is shown in the Figma design.
image

File: src/components/Header/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import main_img from '../.././assets/logo.svg';
+import up_arrow from '../.././assets/up-arrow.svg';
+import git_img from '../.././assets/Vector.svg';
```
Comment (Waishnav) [inline src/components/Header/Header.jsx]: can you change the name of github svg file from `Vector.svg` to `github.svg`

File: src/components/Header/Header.jsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+            <div className="Container_1">
+                <img src={main_img} alt="Watcher-Logo" />
+                <p>
+                    Watcher
+                    <span className="borderBottom"></span>
+                    <img src={up_arrow} alt="" />
+                </p>
+            </div>
+
+            <button>
```
Comment (Waishnav) [inline src/components/Header/Header.jsx]: change the button to link/ anchor tag.
And add href="https://github.com/Waishnav/Watcher"

**Review by Waishnav — CHANGES_REQUESTED** [review]

**Comment by mittalvansh** [comment]
Changes done!

File: src/components/Header/Header.css [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+        align-items: center;
+        margin-right: 8rem;
+    }
+
+    .Container_1 img {
+        margin-left: 3rem;
+    }
+
+    button {
+        display: none;
```
Comment (Waishnav) [inline src/components/Header/Header.css:98]:
```
a .StarUs {
   display: none;
}

**Review by Waishnav — CHANGES_REQUESTED** [review]

File: src/components/Header/Header.css (hunk 5 above)
Comment (Waishnav) [reply src/components/Header/Header.css:98]: Also do change padding-top of Container to 1rem

**Pull request merged**